### PR TITLE
Fix examples2 after conan list changes and remove fixmes

### PR DESCRIPTION
**Pull request by czoido**

### CODE REVIEW

File: tutorial/creating_packages/handle_sources/run_example.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
 output = run("conan list hello/1.0#:*")
 
 # assert that there are 3 binaries in the cache
-assert output.count("PID: ") == 3
```
Comment (franramirez688) [inline tutorial/creating_packages/handle_sources/run_example.py:40]: IMO, all these counts look a little bit fragile. What if we get the JSON file and read it? I think that could be better and more robust. WDYT?

Comment (czoido) [reply tutorial/creating_packages/handle_sources/run_example.py:40]: Yes I think that's much better, maybe we can merge this to have the ci in green and add some function to count packages in the near future?

File: tutorial/creating_packages/handle_sources/run_example.py [hunk 2]
```diff
@@ -37,7 +37,7 @@
...
+assert output.count("arch: ") == 3
```
Comment (czoido) [inline tutorial/creating_packages/handle_sources/run_example.py:40]:
```suggestion
TODO: better do this parsing the json, this is too fragile
assert output.count("arch: ") == 3
```

File: tutorial/creating_packages/handle_sources/run_example.py (hunk 1 above)
Comment (czoido) [reply tutorial/creating_packages/handle_sources/run_example.py:40]: I added a TODO

**Review by franramirez688 — APPROVED** [review]

File: tutorial/creating_packages/handle_sources/run_example.py [hunk 3]
```diff
@@ -37,7 +37,8 @@
 output = run("conan list hello/1.0#:*")
 
 # assert that there are 3 binaries in the cache
-assert output.count("PID: ") == 3
+TODO: better do this parsing the json, this is too fragile
```
Comment (czoido) [inline tutorial/creating_packages/handle_sources/run_example.py:40]:
```suggestion
#TODO: better do this parsing the json, this is too fragile
```

**Pull request merged**